view/theme/frio/templates/admin/users.tpl

```diff
@@ -0,0 +1,272 @@
+<script>
+	function confirm_delete(uname){
+		return confirm( "{{$confirm_delete}}".format(uname));
+	}
+	function confirm_delete_multi(){
+		return confirm("{{$confirm_delete_multi}}");
+	}
+	function selectall(cls){
+		$("."+cls).prop("checked", true);
+		return false;
+	}
+	function unselectall(cls){
+		$("."+cls).prop("checked", false);
+		return false;
+	}
+	function details(uid) {
+		$("#user-"+uid+"-detail").toggleClass("hidden");
+		return false;
+	}
+</script>
+<style>
```

Comment:
please move style information to `style.css` (below https://github.com/friendica/friendica/blob/develop/view/theme/frio/css/style.css#L2937)

OR

put in a file `mod_admin.css` in `view/theme/frio/css` and include it with `<link rel="stylesheet" href="view/theme/frio/css/mod_admin.css" type="text/css" media="screen"/>`